explorer/src/components/BondBreakdown.tsx

```diff
@@ -11,103 +11,144 @@ import Paper from '@mui/material/Paper';
-                        </TableRow>
-                        <TableRow>
-                            <TableCell
-                                sx={{
-                                    width: matches ? '90px' : 'auto',
-                                }}
-                                align='left'
-                            >
-                                Pledge total
-                            </TableCell>
-                            <TableCell align='left'>
-                                {bonds.pledges}{bonds.denom}
-                            </TableCell>
-                        </TableRow>
-                        <TableRow>
-                            <TableCell
-                                sx={{
-                                    width: matches ? '90px' : 'auto',
-                                }}
-                                align='left'
-                            >
-                                Delegation total
-                            </TableCell>
-                            <TableCell align='left'>
-                                {bonds.delegations}{bonds.denom}
-                            </TableCell>
-                        </TableRow>
-                    </TableBody>
-                </Table>
+    React.useEffect(() => {
+        const hasError = mixnodeDetailInfo?.error || delegations?.error ? true : false;
+        const hasAllMixnodeInfo = mixnodeDetailInfo?.data !== undefined && mixnodeDetailInfo?.data[0].mix_node ? true : false;
+        const hasAllDelegationsInfo = delegations?.data !== undefined ? true : false;
+        const hasAllData = !hasError && hasAllMixnodeInfo && hasAllDelegationsInfo ? true : false;
+        setShowError(hasError);
+        setAllContentLoaded(hasAllData);
+    }, [mixnodeDetailInfo, delegations]);
+    
+    const calcBondPercentage = (num: number) => {
+        if (mixnodeDetailInfo?.data !== undefined && mixnodeDetailInfo?.data[0]) {
+            const rawDeligationAmount = Number(mixnodeDetailInfo.data[0].total_delegation.amount);
+            const rawPledgeAmount = Number(mixnodeDetailInfo.data[0].bond_amount.amount);
+            const rawTotalBondsAmount = rawDeligationAmount + rawPledgeAmount;
+            return (num * 100 / rawTotalBondsAmount).toFixed(1)
+        }
+    }
 
-                {delegations?.data !== undefined && delegations?.data[0] && (
-                    <Table sx={{ minWidth: 650 }} aria-label='delegation totals'>
-                        <TableHead>
+    if (mixnodeDetailInfo?.isLoading) {
+        return <p>Loading...</p>
+    }
+
+    if (showError) {
+        return <p>We are unable to retrieve a Mixnode with that ID. Please try later or Contact Us.</p>
+    }
+
+    if (!showError && allContentLoaded) {
+        return (
+            <>
+                <TableContainer component={Paper} sx={{ background: theme => mode === 'dark' ? theme.palette.secondary.dark : theme.palette.primary.light }}>
```

Comment:
I take it this doesnt work without `background: theme => mode === 'dark' ? theme.palette.secondary.dark : theme.palette.primary.light`?

Just curious as I thought using the component={Paper} would have sorted it 

Fine to keep as is.